cinema/tests/test_actor_api.py

```diff
@@ -16,9 +16,7 @@ def test_get_actors(self):
         response = self.client.get("/api/cinema/actors/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         actors_full_names = [actor["full_name"] for actor in response.data]
-        self.assertEqual(
-            sorted(actors_full_names), ["George Clooney", "Keanu Reeves"]
-        )
+        self.assertEqual(sorted(actors_full_names), ["George Clooney", "Keanu Reeves"])
```

Comment:
please don't change tests

Reply:
I'm intresting)

Reply:
okay I understand it was black module and I didnt check git files just push. Thanks!